dss/events/handlers/index.py

```diff
@@ -78,7 +78,7 @@ def create_index_data(handle: BlobStore, bucket_name: str, bundle_id: str, manif
     files_info = manifest[BundleMetadata.FILES]
     index_files = {}
     for file_info in files_info:
-        if file_info[BundleFileMetadata.INDEXED] is True:
+        if file_info[BundleFileMetadata.INDEXED]:
```

Comment:
Why this change?

Reply:
Also, as a pattern, it might be easier to do:

```
for file_info in files_info:
    if file_info[BundleFileMetadata.INDEXED] is not True:
        continue
    do_other_work()
```

Reply:
The first change above was to reduce unnecessary verbosity, yet would be better addressed in a separate PR focused on style clean-up. I'll undo the change and address it in another PR.
The second change will be a nice improvement, and will be addressed in the style PR, yet shouldn't be included in this PR, IMO.